test/integration/test_cli_commands.py

```diff
@@ -874,3 +874,20 @@ def test_logging_with_logformat_logstash_flag(self):
                     obj={"filters":[]})
         output = sorted(result.output.splitlines(), reverse=True)[:4]
         self.assertEqual(expected, output)
+
+class TestCLIRepositoryCreate(CuratorTestCase):
+    def test_create_fs_repository(self):
+        test = clicktest.CliRunner()
+        result = test.invoke(
+                    curator.repomgrcli,
+                    [
+                        '--logfile', os.devnull,
+                        '--host', host,
+                        '--port', str(port),
+                        'create',
+                        'fs',
+                        '--repository', self.args['repository'],
+                        '--location', '/tmp'
```

Comment:
I had to hard-code this as '/tmp' because I am running elasticsearch in a virtual machine and happen to have my ports forwarded (so localhost:9200 is hitting ES on my VM). The test utilities attempt to create a temporary folder, but that folder only exists on my host machine. '/tmp' seemed reasonable, for now.